client/components/section-nav/index.jsx

```diff
@@ -61,7 +62,7 @@ var SectionNav = React.createClass( {
 			className = classNames( {
 				'section-nav': true,
 				'is-empty': true
-			} );
+			}, this.props.className );
```

Comment:
@mtias as this is a meetup project and we don't want to get blocked, we'll merge this into the `add/publicize-scheduling` feature branch and if you reply, we can make adjustments in the feature branch.